docs/tools/cli/byoc.md

```diff
@@ -61,6 +61,71 @@ Modifies a custom cloud configuration.
 | `--reserved-cidr`   | No       | IP address range of the VPC in your cloud account for Aiven services created in the custom cloud                        |
 | `--display-name`    | No       | Name of the custom cloud                                                                                                |
 
+## Tag a custom cloud
+
+### `avn byoc tags list`
+
+Returns infrastructure tags attached to a custom cloud.
+
+**Syntax**
+
+```bash
+avn byoc tags list                              \
+    --organization-id "ORGANIZATION_IDENTIFIER" \
+    --byoc-id "CUSTOM_CLOUD_IDENTIFIER"
+```
+
+**Parameters**
+
+| Parameter           | Required | Information                                                                                                             |
+| ------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where to modify the custom cloud                                                          |
+| `--byoc-id`         | Yes      | Identifier of the custom cloud to be modified                                                                           |
+
+### `avn byoc tags update`
+
+Adds, updates, or removes infrastructure tags on a custom cloud.
```

Comment:
I think we should explain that these tags are cascaded to the Bastion nodes and disks.